Assets/Minigames/ShapeMinigame/ShapeMinigame.cs

```diff
@@ -145,6 +146,7 @@ void Awake()
     protected override void Start()
     {
         base.Start();
+        minigameShapeController.SetCanMoveShapes(takeInput);
         GetSolution();
         PlaceShapePrefabs();
         SetCamera();
```

Comment:
Moving `SetCamera` over to the `Minigame` class is sensible, but calling it should probably also happen is the abstract `Minigame` class. Same goes for `GetSolution`.